server/RateLimiter.ts

```diff
@@ -0,0 +1,73 @@
+import { isEmpty } from "lodash";
+import { RateLimiterRedis } from "rate-limiter-flexible";
+import env from "@server/env";
+import Logger from "@server/logging/Logger";
+import Redis from "@server/redis";
+import { RateLimiterConfig } from "@server/types";
+
+export default class RateLimiter {
+  constructor() {
+    throw Error(`Cannot instantiate class!`);
+  }
+
+  static readonly rateLimiterMap = new Map<string, RateLimiterRedis>();
+  static readonly defaultRateLimiter = new RateLimiterRedis({
+    storeClient: Redis.defaultClient,
+    points: env.RATE_LIMITER_REQUESTS,
+    duration: env.RATE_LIMITER_DURATION_WINDOW,
+    keyPrefix: env.RATE_LIMITER_REDIS_KEY_PREFIX,
```

Comment:
No need for this to be in `env`, it doesn't make sense to configure it separately. Just a private const on RateLimiter would be fine

Reply:
Lets set the rate limiter window to 60s too, much easier to reason about